### PR TITLE
Add Slack channel invite link

### PR DESCRIPTION
**Pull request by amitkumarj441**
Signed-off-by: Amit Kumar Jaiswal <amitkumarj441@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/community/13)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,8 @@ mailing list.
 
 Slack server: [kubeflow.slack.com](https://kubeflow.slack.com/)
 
+[Join Slack Channel](https://join.slack.com/t/kubeflow/shared_invite/enQtMjgyMzMxNDgyMTQ5LWUwMTIxNmZlZTk2NGU0MmFiNDE4YWJiMzFiOGNkZGZjZmRlNTExNmUwMmQ2NzMwYzk5YzQxOWQyODBlZGY2OTg)
```
Comment (jlewi) [inline README.md]: Where did you get this link from? Does the link expire?

Comment (amitkumarj441) [reply README.md]: @jlewi I didn't remember actually, but this is the link I saved in my Google doc when I joined Kubeflow and this link works for me.

Comment (ScorpioCPH) [reply README.md]: @ewilderj Do you have any new invite links?

Comment (ewilderj) [reply README.md]: @ScorpioCPH nope, I am not an admin on the Slack server.

Comment (jlewi) [reply README.md]: @amitkumarj441 Can you do a search to see what it takes make slack open so anyone can join? IIRC you actually have to use some third party app to auto-generate invites.

Comment (amitkumarj441) [reply README.md]: @jlewi I can get a auto-generated invite link (I already have one for Rust India community longtime here: http://rustindia.herokuapp.com/) for anyone to join and yes, Thanks for reminding me about 3rd party apps :+1: :smile: 
Pinging you on slack to get this done now and will update here.

**Comment by jlewi** [comment]
I think based on our conversation on slack additional work is needed and this isn't ready but I wanted to double check you aren't waiting on me.

**Pull request closed without merging**